styles.css

```diff
@@ -0,0 +1,233 @@
+:root {
+  --primary: #172b4d;
+  --h-color: #3a4a42;
+  --secondary: #36b37e;
+  --secondary-btn: #36b37f;
+  --weight: bold;
+  --background: #ebf0ee;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  overflow-x: hidden;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.logo {
+  text-decoration: none;
+  font-size: 1.6rem;
+  color: #28352f;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  font-family: "Inter", sans-serif;
+  overflow-x: hidden;
+  line-height: 1.5;
+  -webkit-font-smoothing: antialiased;
+}
+
+header {
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: #fff;
+}
+
+header h2 {
+  animation: crane-float 2s alternate 5 ease-in-out both;
+}
+
+nav .nav-menu {
```

Comment:
- [ ] If you want to target an HTML element by its class, you can just use the dot notation. `.nav-menu` in this case, the `nav` is not necessary.